src/OpenAL/OpenToolkit.OpenAL/Native/ApiContainer.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using OpenToolkit.Core.Loader;
+
+namespace OpenToolkit.Audio.OpenAL
+{
+#pragma warning disable SA1649 // File name should match first type name
```

Comment:
Whats wrong with renaming the file here?

Reply:
It's just some mess I've created, will be fixed before merge :)